pkg/sc/websiteStorer.go

```diff
@@ -1,7 +1,7 @@
 // GENERATED BY textFileToGoConst
 // GitHub:     github.com/logrusorgru/textFileToGoConst
-// input file: sc/websiteStorer.wasm
-// generated:  Tue Aug  9 17:21:32 CEST 2022
+// input file: sc\websiteStorer.wasm
```

Comment:
painful. Maybe we can ask textFileToGoConst to avoid setting a date and the input file path to avoid having a diff ?

Reply:
The problem is that it's not configurable : 
https://github.com/logrusorgru/textFileToGoConst/blob/c0d549f6eab77c361ef3305865e3571f8d3a1198/main.go#L152

Reply:
Cool, I've found why this project was stopped: we should use https://pkg.go.dev/embed@master. In a new issue ?

Reply:
Issue created

https://github.com/massalabs/thyra/issues/70